### DISCUSSION TITLE
Some new information about 3 table objects! (Enums, MultiLanguage and Obj)

### DISCUSSION THREAD

**Discussion by lukas-runge** [hossimo/GMA3Plugins#29] [Show and tell]
**Hi everyone,**

I just noticed that 3 of the `_G` Variables from grandMA3 Lua API are not functions, but tables. I just [added](https://github.com/hossimo/GMA3Plugins/wiki/_compare/b173c176947579f38923e5b57f8d44cafc76a6ca...0e641f55d1c00eba8d1ee563153e682c05fbdccc?diff=split) all knowledge I was able to gather about them to the [wiki](https://github.com/hossimo/GMA3Plugins/wiki):

- [Enums](https://github.com/hossimo/GMA3Plugins/wiki/Enums)
- [MultiLanguage](https://github.com/hossimo/GMA3Plugins/wiki/MultiLanguage)
- [Obj](https://github.com/hossimo/GMA3Plugins/wiki/Obj)

**What could those Objects be for? What may we be able to do with them?**
Happy to hear your ideas! 🌟 

Kind regards
@lukas-runge

**Comment by hossimo** [comment]
That's how I originally came up with the list on the home page, something like this:

``` lua
    local file, err = io.open(exportFilePath .. "globals.txt", "w+")

    if file then
        file:write(os.clock() .. "\n")
        local count = 1
        for k, v in pairs(_G) do
            file:write(string.format("%s\n",  k))
            count = count + 1
        end
        io.close(file)
        Printf("Wrote Globals to %s", exportFilePath)
        SyncFS();
    end
```

I do have another function that iterates through classes, but I have to guess at their properties and methods, but sometimes you can find more info by going through the console structure and listing objects that look like they belong.

I haven't have much time lately but I hope to get back to it soon.
